guides/plugins/plugins/bundle.md

```diff
@@ -0,0 +1,109 @@
+---
+nav:
+  title: Bundle
+  position: 20
+
+---
+
+# Bundle
+
+Plugins are based on the Symfony bundle concept, but offer additional features like lifecycle events and the ability to be managed in the Shopware administration. This is maybe unwanted in some cases, like project critical customizations which should not be managed via the Shopware administration. In this case, you can use a Symfony bundle instead of a plugin.
+
+The Bundle can be put into the `src/` folder of your project and needs to be registered in the `config/bundles.php` file of your project.
```

Comment:
Can we link to something, on how a project folder looks like or a guide, where we set up a project?